spec/services/reporting/account_reuse_report_spec.rb

```diff
@@ -61,57 +129,72 @@
       # User 6 has 2 SPs and only 1 shows up in the query
       # User 7 has 1 SP and 1 shows up in the query
       # User 8 has 1 SP and 0 show up in the query
+      # User 9 has 2 SPs and only 1 shows up in the query
+      # User 10 has 2 SPs and 0 show up in the query
+      # User 11 has 2 SPs and 0 show up in the query
+      # User 12 has 1 SP and 1 shows up in the query
+      # User 13 has 1 SP and 0 show up in the query
+      # User 14 has 1 SP and 0 show up in the query
+      # User 15 has 12 SPs and 12 show up in the query
+      # User 16 has 11 SPs and 11 show up in the query
+      # User 17 has 11 SPs and 11 show up in the query
+      # User 18 has 10 SPs and 10 show up in the query
+      # User 19 has 10 SPs and 10 show up in the query
+      # User 20 has 10 SPs and 9 show up in the query
       #
       # This will give 1 user with 3 SPs/apps and 3 users with 2 SPs/apps for the IDV app report
       # This will give 4 users with 3 SPs/apps and 5 users with 2 SPs/apps for the ALL app report
-      # This will give 3 users with 2 agencies for the IDV agency report
-      # This will give 7 users with 2 agencies for the ALL agency report
+      # This will give 1 user with 9 SPs/apps for the ALL app report and 0 for the IDV app report
+      # This will give 6 users with 10-12 SPs/apps for the ALL app report
+      # This will give 5 users with 10-12 SPs/apps for the IDV app report
+      # This will give 9 users with 2 agencies for the IDV agency report
+      # This will give 13 users with 2 agencies for the ALL agency report
 
       users_to_query = [
         { id: 1, # 3 apps, 2 agencies
           created_timestamp: in_query,
-          sp: [sp_a, sp_b, sp_c],
-          sp_timestamp: [in_query, in_query, in_query] },
+          sp: all_agency_apps[0..2],
+          sp_timestamp: Array.new(3) {in_query} },
         { id: 2, # 3 apps, 2 agencies
           created_timestamp: in_query,
-          sp: [sp_a, sp_b, sp_c],
-          sp_timestamp: [in_query, in_query, out_of_query] },
+          sp: all_agency_apps[0..2],
+          sp_timestamp: Array.new(2) {in_query} + Array.new(1) {out_of_query} },
```

Comment:
1. this is going to have rubocop issues with spacing inside the curly braces
2. for only 3 timestamps, I think a literal array might be clearer
```suggestion
          sp_timestamp: [in_query, in_query, out_of_query] },
```